src/components/ModelizerModelView.vue

```diff
@@ -58,10 +55,10 @@ const data = reactive({
 async function getFileInputs(plugin, fileInformations) {
   return Promise.allSettled(
     fileInformations
-      .filter((fileInfo) => plugin.parser.isParsable(fileInfo))
+      .filter((fileInfo) => plugin.isParsable(fileInfo))
       .map((fileInfo) => readProjectFile(props.projectName, fileInfo)),
   ).then((allResults) => allResults
-    .filter((result) => result.status === 'fulfilled')
+    .filter((result) => result.status === 'fulfilled' && result.value.content !== null)
```

Comment:
what was the case that made you add this condition ?

Reply:
done